docker/push.sh

```diff
@@ -19,9 +19,7 @@ TAG=${2:-beta}
 
 echo "Building AcousticBrainz web image with env $ENV tag $TAG..."
 docker build -t metabrainz/acousticbrainz:$TAG \
-        --target acousticbrainz-prod \
-        --build-arg GIT_COMMIT_SHA=$(git rev-parse HEAD) \
-        --build-arg deploy_env=$ENV .
```

Comment:
now that we have no env-specific startup scripts, we don't need this. Not sure if we still need the git data - it would be useful in sentry, but we don't install it

Reply:
We should definitely try to install it!

Reply:
https://tickets.metabrainz.org/projects/BU/issues/BU-31